db/python/layers/participant.py

```diff
@@ -621,3 +621,17 @@ async def update_participants(
         # pylint: disable=unused-argument
         d = {k: v for k, v in locals().items() if k != 'self'}
         return await self.pttable.update_participants(**d)
+
+    async def update_single_participant(
+        self,
+        participant_id: int,
+        reported_sex: int = None,
+        reported_gender: str = None,
+        karyotype: str = None,
+        meta: Dict = None,
+        author=None,
+    ):
```

Comment:
By removing the project earlier, can you add a `check_project_id=True` + the actual check before we do the update participant.